### ISSUE TITLE
Add support for adding updatable platform-specific Kubernetes manifests 

### ISSUE THREAD

**Issue by invidian** [kinvolk/lokomotive#478]
To resolve #467, we need to add platform-specific GNPs to block access to metadata servers on cloud platforms. Currently, we don't have a good way to manage those manifests.

**Comment by johananl** [comment]
One simple solution could be to block access to all metadata endpoints on all platforms. I don't see any harm in disallowing access to `169.254.169.254` on Packet for example. Would that work?

**Comment by rata** [comment]
I agree with @johananl for AWS if we need that ASAP (happen to comment the same here: https://github.com/kinvolk/lokomotive/pull/464#issuecomment-633577005). But might not work for other platforms.

I think currently we might not be able to add platform-specific charts on control-plane chart, but that seems reasonable IMHO. I'm not familiar with the helm part to know if that reasonable or even something we want.

Lokomotive component seems okay too. And we can do the refactoring to have default lokomotive components our of scope for this
